tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -193,6 +194,25 @@ Verify JupyterHub Pod Logs Dont Have Errors About Distutil Library
     ...       ODS-586
     Verify Errors In Jupyterhub Logs
 
+Verify Grafana Is Connected To Prometheus Using TLS
+    [Documentation]    Verifies Grafana is connected to Prometheus using TLS
+    [Tags]    Tier2
+    ...       ODS-963
+    [Setup]  Set Library Search Order  Selenium Library
+    ${res} =  Oc Get  kind=Secret  name=grafana-datasources  namespace=redhat-ods-monitoring
+    ${decode} =  Evaluate  base64.b64decode("${res[0]['data']['datasources.yaml']}").decode('utf-8')  modules=base64
+    ${decode} =  Evaluate  json.loads('''${decode}''')  json
+    Run Keyword If  'tlsSkipVerify' in ${decode['datasources'][0]['jsonData']}
+    ...  Should Be Equal As Strings  ${decode['datasources'][0]['jsonData']['tlsSkipVerify']}  False
```

Comment:
these 2 lines could be wrapped into "Check Tls Is Enabled"

Reply:
Updated.